src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -122,13 +140,13 @@ protected void execute(final Event e) {
 					if (script != null)
 						scriptName = script.getConfig().getFileName();
 				}
-				Skript.info("[" + scriptName + "] " + message);
+				SkriptLogger.LOGGER.log(logLevel, "[" + scriptName + "] " + messages);
 			}
 		}
 	}
-	
+
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		return "log " + messages.toString(e, debug) + (files != null ? " to " + files.toString(e, debug) : "");
+	public String toString(@Nullable Event e, boolean debug) {
+		return "log " + messages.toString(e, debug) + (files != null ? " to " + files.toString(e, debug) : "") + logLevel;
```

Comment:
Needs updated to better reflect the log level syntax. If logLevel is INFO, append nothing. Otherwise, append something like `with a severity of <LEVEL>`

Reply:
oh shoot you're right, I will look into that